project-1/gspread_code.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Oct 16 15:12:29 2023
+
+@author: computer world
+
+Project : getting data from github repositories(through selenium library) and 
+sending to google spreadsheet with gspread package
+"""
+
+import gspread
+import pandas as pd
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+path = "documents/datasets_for_data_science/gspread_project/spartan-figure-402211-2241d0349aab.json"
+
+def update_spreadsheet(spreadsheet = "Gspread SpreadSheet",worksheet="Sheet1"):
+    
+    client = gspread.service_account(filename=path)
+    worksheet = client.open(spreadsheet).worksheet(worksheet)
+    worksheet.clear() #clearning worksheet on each run of program
```

Comment:
add comments above the line you intend to speak on

Reply:
Just git add, commit and push again

Reply:
alright thanks sir